src/components/Badge/index.tsx

```diff
@@ -58,7 +58,8 @@ const generateStyles = ({
       };
       background: ${
         theme.colors[sentiment][background as keyof typeof theme.colors.primary]
-      }
+      };
+      border: 1px solid transparent;
```

Comment:
What about we use same color for border? There might be a size different for neutral as it has 1px more into its border

```suggestion
      border: 1px solid ${theme.colors[sentiment][background as keyof typeof theme.colors.primary]};
```